packages/dids/src/did-resolver.ts

```diff
@@ -99,9 +104,41 @@ export class DidResolver {
         didUrl: parsedDid.did,
         resolutionOptions
       });
+
       await this.cache.set(parsedDid.did, resolutionResult);
 
       return resolutionResult;
     }
   }
+
+  async deference(params: DereferenceParams): Promise<DidResource> {
```

Comment:
Hey @mistermoe -- haven't reviewed this PR in detail since its still in draft but noticed one small typo

```suggestion
  async dereference(params: DereferenceParams): Promise<DidResource> {
```

Reply:
ah thank you! 1am mistakes